Gordon360/Models/CCT/Housing/RA_Task_Status.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("Ra_ID", "Task_ID")]
+[Table("RA_Task_Status", Schema = "Housing")]
+public partial class RA_Task_Status
+{
+    [Key]
+    public int Ra_ID { get; set; }
+
+    [Key]
+    public int Task_ID { get; set; }
+
+    public bool? Status { get; set; }
```

Comment:
What does `Status` mean in this context? It's not clear what a status of `true` means - is `true` a complete task, or an incomplete task? For this kind of boolean, it's usually better to give a descriptive name like `is_Completed`, or even `complete` if you really want to keep the column name short.